packages/dds/tree/src/tree/anchorSet.ts

```diff
@@ -430,6 +433,17 @@ class PathNode implements UpPath {
         public readonly anchorSet: AnchorSet,
         public parentField: FieldKey,
         public parentIndex: number,
+        /**
+         * Starts as `undefined` for the {@link AnchorSet.root}.
+         * Starts as defined but is changed to `undefined` when the corresponding document node is
```

Comment:
This comment seems to disagree with itself for how this starts.

Reply:
Is it because I don't explicitly say that the second like only applies when the first doesn't?
I tweaked it. Let me know if that's clearer.

Reply:
Its unclear if "Starts as `undefined` for the {@link AnchorSet.root}." is saying this starts as undefined, meaning that the parent of this is the root (aka app path nodes start under the root), OR if it starts as undefined if this node is the root, or it starts is undefined if this node starts under the root.

Either way I think documenting what a value starts at (all values in JS start as undefined really...) is a lot less useful that documenting what it means (more like whats you documented below). Maybe something like:

> The parent of this PathNode (an up pointer in the PanthNode tree).
> There must be a corresponding down pointer from the parentPath to this.
> When undefined, this node is the {@link AnchorSet.root} for `this.anchorSet` and thus has no parent.
> 
> When updating the tree, AnchorSet may transiently leave the up and down pointers inconsistent (updating down pointers first), but must ensure they are consistent before the editing operation returns to non AnchorSet code.

We document that you can't access dead or disposed nodes (on the state enum values), so there is no reason for us to set of documents what this field does in either of those cases (accessing this field would be invalid).

Reply:
Thanks, that is indeed much better. Done.